src/openms/thirdparty/IsoSpec/IsoSpec/isoSpec++.cpp

```diff
@@ -437,7 +439,9 @@ Lcutoff(_threshold <= 0.0 ? std::numeric_limits<double>::lowest() : (_absolute ?
             empty = true;
     }
 
-    maxConfsLPSum[0] = marginalResults[0]->getModeLProb();
+    if(dimNumber > 1)
+        maxConfsLPSum[0] = marginalResults[0]->getModeLProb();
```

Comment:
great, will this fix our calculations for the C100 molecule and the issue we had with different compilers?

Reply:
That definitely looks like it could be the underlying issue causing this. I'll have a look into this.

Reply:
okay, it seems to work with GCC, but I can't seem to get the OpenMS to compile with clang, so I can't really test it...